### ISSUE TITLE
gradle

### ISSUE THREAD

**Issue by yigit** [yigit/android-priority-jobqueue#13]
_From @yigit on November 6, 2013 19:50_

move building & testing to gradle.
push artifact as soon as project becomes public.

_Copied from original issue: path/android-priority-jobqueue#7_

**Comment by yigit** [comment]
_From @stefanhoth on July 8, 2014 9:37_

I think this ticket is obsolete. Good. :smile:

**Comment by mseshachalam** [comment]
when will this fork get a new groupid in maven ?

**Comment by yigit** [comment]
when i release the tagging (potentially w/ cancelation since it is the first customer of tagging).

btw, i'm considering jcenter since android is released there. I remember setting up maven central was painful, i wonder if it is easier in jcenter.

**Comment by yigit** [comment]
I already created a version under group name com.birbit ; using maven central (ossrh).
snapshot for 1.3 (tagging) should be public in 1-2 days.

**Comment by mseshachalam** [comment]
https://bintray.com/yigit/maven/com.birbit.android/
Does it include the cancellation of jobs ?

**Comment by yigit** [comment]
yes it does. just lacking some more tests to release.

yigit

**Comment by cjhenck** [comment]
I think that the transitive dependencies should perhaps be removed (or updated).  I get a lot of warnings about commons-logging, which is included through com.google.android:android:2.2.1 within the android-priority-jobqueue package:

```
WARNING: Dependency commons-logging:commons-logging:1.1.1 is ignored for globalDebug as it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage it with jarjar to change the class packages
```

**Comment by yigit** [comment]
You are right, i should just remove android dependency since it is useless in this scope.

**Comment by yigit** [comment]
fixed in 1.3.2
